apps/web/app/(routes)/page.tsx

```
@@ -29,13 +31,29 @@ export default function Home() {
     }
   }, [servers]);
 
+
+  useEffect(() => {
+    async function fetchServers() {
+      setTagLoading(true);
+      const data = await fetch(TAGS_API_ENDPOINT);
+      const response = await data.json();
+      setTags([...response?.tags, ...tags]);
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)